Dumplings/Stats/Statista.cs

```diff
@@ -1377,6 +1377,46 @@ public void CalculateAndUploadMonthlyCoinJoins()
             return myDic;
         }
 
+        public Dictionary<YearMonthDay, decimal> CalculateDailyFriendsDontPayAmount()
+        {
+            var postMixTxHashes = ScannerFiles.Wasabi2PostMixTxHashes;
+            var ww2CoinJoins = ScannerFiles.Wasabi2CoinJoins;
+            var myDic = new Dictionary<YearMonthDay, decimal>();
+
+            foreach (var tx in ww2CoinJoins)
+            {
+                var blockTime = tx.BlockInfo.BlockTime;
+                if (blockTime.HasValue)
+                {
+                    var blockTimeValue = blockTime.Value;
+                    var yearMonthDay = new YearMonthDay(blockTimeValue.Year, blockTimeValue.Month, blockTimeValue.Day);
+
+                    decimal sum = 0;
+                    foreach (var input in tx.Inputs.Where(x => postMixTxHashes.Contains(x.OutPoint.Hash)))
```

Comment:
The complexity of this seems crazy. What is the usual size of `postMixTxHashes`?  It seems better to use a `Dictionary` and `ContainsKey`